docs-site/content/0.19.0/guide/features/federated.md

```diff
@@ -0,0 +1,182 @@
+# Federated Search
+
+Federated or multi search is a way to search for documents in multiple collections as part of a single search query. Federated search can help reduce network latencies. It can also be used to present similar content from other collections, that might encourage users to browse more content across your application. For example, your application might have different collections for `Nike` and `Adidas`. Now, if a user is looking for a shoe from a specific branch and they might not know what other brands are available, the search query can perform a search on both the collections and return relevant results from both the collections. 
```

Comment:
> Federated or multi search is a way to search for documents in multiple collections as part of a single search query.

[Add] ... You can also use multi-search to send multiple search queries to the same collections, essentially giving you a way to batch search queries in a single HTTP request.

Reply:
To be able to get a screenshot, could we use the example of searching for say "Canon" on this site: https://www.bhphotovideo.com/

It should show results across products, help articles and search suggestions.